src/components/dapp-staking-v2/my-staking/components/CardList.vue

```diff
@@ -0,0 +1,257 @@
+<template>
+  <div class="wrapper--list">
+    <div
+      v-for="(t, index) in dapps"
+      :key="t.contract.address"
+      class="card"
+      @mouseover="hoverIndex = index"
+      @click="goDappPageLink(t.dapp?.address)"
+    >
+      <div class="wrapper--card">
+        <div class="wrapper--img">
+          <q-img :src="t.dapp?.iconUrl" class="img--dapp" fit="contain" no-spinner />
+        </div>
+        <div class="panel--right">
+          <div class="txt--title">{{ t.dapp?.name }}</div>
+          <div class="badge--tag">{{ category }}</div>
+          <div class="divider"></div>
+          <div class="row--metric">
+            <div class="row--numStakers">
+              <astar-icon-base class="icon--community" stroke="currentColor" width="20" height="18">
+                <astar-icon-community />
+              </astar-icon-base>
+              {{ t.stakerInfo.stakersCount.toLocaleString() }}
+            </div>
+            <div>
+              <token-balance
+                :balance="t.stakerInfo.totalStakeFormatted.toString()"
+                :symbol="nativeTokenSymbol"
+                :decimals="0"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <astar-button
```

Comment:
[Screencast from 11-02-2022 12:31:11 PM.webm](https://user-images.githubusercontent.com/40356749/199479351-33b2e792-85aa-41e1-a04c-2e3dffc29b98.webm)

The button will stake visible even though the item highlight disappears when the user hovers over an item. Is this on purpose?

Reply:
I am not sure about this. @sirius651 do you maybe know?

Reply:
Good catch. fixed [this](https://github.com/AstarNetwork/astar-apps/pull/523/commits/e5ec88574049bd7a15bf03447341c3a2a922258b) quickly